fe/fe-core/src/main/java/com/starrocks/sql/optimizer/dump/DumpInfo.java

```diff
@@ -42,6 +42,9 @@ default void addView(String dbName, View view) {
     default void addTableStatistics(Table table, String column, ColumnStatistic columnStatistic) {
     }
 
+    default void addTableHistogramStatistics(Table table, String column, ColumnStatistic columnStatistic) {
```

Comment:
you'd better add [multi-column](https://github.com/StarRocks/starrocks/blob/main/fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/CachedStatisticStorage.java#L79) stats to query dump.

Reply:
sure